tests/chaos/conftest.py

```diff
@@ -186,25 +187,34 @@ def rebooting_master_node(
 
 @pytest.fixture()
 def pod_deleting_process(request, admin_client):
+    """
+    A pytest fixture to create and manage a pod-deleting process during a test.
+    This fixture sets up a process to delete pods matching a specific prefix in a given namespace
+    with specified parameters. The process starts before the test begins and ensures proper
+    cleanup after the test ends.
+    Args:
+        request (SubRequest): Pytest's request object that provides access to the parameters passed to the fixture.
+        admin_client (DynamicClient): The Kubernetes dynamic client used to interact with the cluster.
+    Returns:
+        PodDeletingProcess: A dataclass instance containing the process object, the namespace name,
+        and the pod prefix, allowing easy access to these details in the test.
+    """
+    PodDeletingProcess = namedtuple("PodDeletingProcess", ["process", "namespace_name", "pod_prefix"])
+
     pod_prefix = request.param["pod_prefix"]
     namespace_name = request.param["namespace_name"]
-    pod_deleting_process = create_pod_deleting_process(
+    process = create_pod_deleting_process(
```

Comment:
one note regarding function _create_pod_deleting_process_ - as I see it uses Multiprocessing. Keep in mind that pytest+multiprocessing potentially may hang the test run forever..

Reply:
For chaos, we need ability to run commands in parallel.

Reply:
yep, I know
I just wanted to warn about possible issues with test run. May be multiprocessing can be changed to threadings (another PR is ok)

Reply:
I'll submit another PR if such an issue happens.